<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnP1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #000000;
    color: #000000;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    font-family: 'Cooper Black'; 
    border-radius: 20px;
}

QPushButton:hover {
    background-color: #cdcfcc;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}
</string>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnP2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #000000;
    color: #000000;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    font-family: 'Cooper Black'; 
    border-radius: 20px;
}

QPushButton:hover {
    background-color: #cdcfcc;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}
</string>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5MIN">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #000000;
    color: #000000;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    font-family: 'Cooper Black'; 
    border-radius: 10px;
}

QPushButton:hover {
    background-color: #cdcfcc;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}
</string>
    </property>
    <property name="text">
     <string>5 min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartGame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #000000;
    color: #000000;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    font-family: 'Cooper Black'; 
    border-radius: 20px;
}

QPushButton:hover {
    background-color: #cdcfcc;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopGame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #000000;
    color: #000000;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    font-family: 'Cooper Black'; 
    border-radius: 20px;
}

QPushButton:hover {
    background-color: #cdcfcc;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}
</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="PB_P1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid #000000;
    border-radius: 12px; 
    background-color: #e8e8e8;
    text-align: center; 
    color: #000000;
    font-size: 30px;
    font-family: 'Cooper Black';
}

QProgressBar::chunk {
    border-radius: 10px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #333333, stop:1 #cccccc); 
    color: #000000;
}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="PB_P2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid #000000;
    border-radius: 12px; 
    background-color: #e8e8e8; 
    text-align: center; 
    color: #000000;
    font-size: 30px;
    font-family: 'Cooper Black';
}

QProgressBar::chunk {
    border-radius: 10px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #333333, stop:1 #cccccc); 
    color: #000000;
}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: #e8e8e8;
    color: #000000; 
    font-family: 'Cooper Black'; 
    border: 2px solid #000000;
    border-radius: 10px; 
}
</string>
    </property>
    <property name="text">
     <string>Chess clock</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2MIN">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #000000;
    color: #000000;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    font-family: 'Cooper Black'; 
    border-radius: 10px;
}

QPushButton:hover {
    background-color: #cdcfcc;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}
</string>
    </property>
    <property name="text">
     <string>2 min</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGameState">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>199</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: #e8e8e8;
    color: #000000; 
    font-family: 'Cooper Black'; 
    border: 2px solid #000000;
    padding: 10px 30px; 
    border-radius: 12px; 
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
